fog/load_testing/src/bin/ingest.rs

```diff
@@ -290,37 +309,38 @@ fn load_test(ingest_server_binary: &Path, test_params: TestParams, logger: Logge
 
         // Measure process_txs load
         {
-            // How many txos we add at a time
-            const CHUNK_SIZE: usize = 250;
-            // How many repetitions we do
-            const REPETITIONS: usize = 100;
-
-            log::info!(logger, "Generating {} random blocks", REPETITIONS);
+            log::info!(logger, "Generating {} random blocks", config.repetitions);
             let num_blocks = ledger_db.num_blocks().unwrap();
-            let last_block = ledger_db.get_block(num_blocks - 1).unwrap();
+            let mut last_block = ledger_db.get_block(num_blocks - 1).unwrap();
             assert_eq!(
                 last_block.cumulative_txo_count,
                 ledger_db.num_txos().unwrap()
             );
 
-            let results = get_blocks(
-                block_version,
-                REPETITIONS,
-                CHUNK_SIZE,
-                1,
-                CHUNK_SIZE,
-                1 << 20,
-                last_block,
-                rng,
-            );
+            let mut results = Vec::default();
+            for _ in 0..config.repetitions {
+                let result1 = get_blocks(
+                    block_version,
+                    1,                 // num_blocks
```

Comment:
:question: What was the reason to not use "repetitions" for the `num_blocks`?
Is it to get new random account keys?

Reply:
The reason was that it was extremely slow at one point and I wanted to get a line of debugging output each time it made a block, so that I had some sense of progress.